config.yaml

```diff
@@ -0,0 +1,42 @@
+# Observations
```

Comment:
Draft header for this file, since I was a bit confused. But not sure if this level of detail is necessary.

# -----------------------------------------------------------------------------------------#
# Configuration file for satellite operators
# This file describes the structure of the satellite or model files used as inputs for the 
# satellite operator. 
#
# Satellite files:
# The description of the satellite files requires the following fields to be defined:
#     - AVERAGING_KERNEL_USES_CENTERS_OR_EDGES: A string with value 
#        'centers' or 'edges' that corresponds to whether the satellite averaging
#        kernel values are measured at the pressure centers or edges of the vertical
#        grid.
#     - PARSER: A string corresponding to the name of a parsing function defined in
#        parsers.py. If new satellites are added, new parsers need to be written.
#    -  DATA_FIELDS: These fields all define strings corresponding to the variable
#        name in the satellite file netcdfs.
#        - N_OBS: The observation dimension (number of observations)
#        - N_EDGES: The dimension containing the vertical level edges
#        - N_CENTERS: The dimension containing the vertical level centers
#        - Other variable values are as described by the yaml fields.
#
# Model files:
# The description of the model files requires the following fields to be defined:
#     - LEVEL_EDGE_FILE_FORMAT: A regex string giving the format of the level edge 
#        file names
#     - CH4_FILE_NAME: A regex string giving the format of the methane output files
#     - PRESSURE_EDGES: A string corresponding to the netcdf variable containing
#        the pressure edges
#     - CH4_CENTERS: A string corresponding to the netcdf variable containing the 
#        methane concentrations at the center of each pressure level
#     - LATITUDE, LONGITUDE, and TIME: Strings corresponding to the netcdf dimensions
#        for each of the corresponding quantities
# -----------------------------------------------------------------------------------------#